src/select/Select.ts

```diff
@@ -70,6 +72,15 @@ export default class Select extends SelectBase<SelectProperties> {
 	private _open = false;
 	private _baseId = uuid();
 
+	private _getOptionLabel(option: any) {
```

Comment:
Shouldn't this be protected without the `_`?

Reply:
Okay, I see that `getOptionLabel` is still passed as a property. Should that be moved to this method instead?

Reply:
@pottedmeat I think the property should stay for consistency with Listbox and ComboBox. I added `_getOptionLabel` because I realized I was doing `getOptionLabel ? getOptionLabel(option) : fallback` a number of times, and wanted to consolidate. I made it private because I don't think it needs to be overridden in addition to having `getOptionLabel`

Reply:
But perhaps it should be public for consistency as well? It's pretty similar to other methods that are public /shrug